slu-scripts/src/sensor-data/tests/sensor-data.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { encryptData } from '../../auth-proof/auth-proof';
+import { sendData } from '../sensor-data';
+import { ChannelClient, ChannelData } from 'iota-is-sdk';
+import fs from 'fs';
+
+const keyFilePath: string | undefined = process.env.npm_config_key_file;
+const inputData: string | undefined = process.env.npm_config_input;
+const destination: string | undefined = process.env.npm_config_dest;
+const encryptedDataPath: string | undefined = process.env.npm_config_input_enc;
+const isConfiguration: string | undefined = process.env.npm_config_config;
```

Comment:
this might should be named is_config instead of just config

Reply:
please rename isConfiguration too to isConfigFile or isConfigPath to demonstrate we have a path here